websites/A/AnizmTV/presence.ts

```diff
@@ -21,135 +22,175 @@ presence.on('iFrameData', async (msg: unknown) => {
 presence.on('UpdateData', async () => {
   const presenceData: PresenceData = {
     largeImageKey: 'https://cdn.rcd.gg/PreMiD/websites/A/AnizmTV/assets/logo.png',
+    startTimestamp: browsingTimestamp, // Add timestamp by default for all states
   }
-  const title = document.querySelector(
-    'html > body > main > #pageContent > div > h2 > a',
-  )
-  const episode = document.querySelector(
-    'html > body > main > #pageContent > div > h2.anizm_pageTitle > span',
-  )
-  const animeSeries = document
-    .querySelector('#pageContent > div > h2 > a')
-    ?.getAttribute('href') || document.URL
 
+  // Get basic selectors once
+  const title = document.querySelector('html > body > main > #pageContent > div > h2 > a')
+  const episode = document.querySelector('html > body > main > #pageContent > div > h2.anizm_pageTitle > span')
+  const animeSeries = document.querySelector('#pageContent > div > h2 > a')?.getAttribute('href') || document.URL
```

Comment:
For consistency across activities, use `document.location.href` instead of `document.URL`